openedx/core/djangoapps/content_libraries/tasks.py

```diff
@@ -557,7 +558,9 @@ def backup_library(self, user_id: int, library_key_str: str) -> None:
         timestamp = datetime.now().strftime("%Y-%m-%d-%H%M%S")
         filename = f'{sanitized_lib_key}-{timestamp}.zip'
         file_path = os.path.join(root_dir, filename)
-        create_lib_zip_file(lp_key=str(library_key), path=file_path)
+        user = User.objects.get(id=user_id)
+        origin_server = settings.CMS_BASE if hasattr(settings, 'CMS_BASE') else None
```

Comment:
Nit: This is equivalent to:


```suggestion
        origin_server = getattr(settings, 'CMS_BASE', None)
```